core/example/longrange/CMakeLists.txt

```diff
@@ -1,13 +1,13 @@
 if(NOT Cabana_ENABLE_MPI)
   return()
 endif()
-  
+
 add_executable(Direct directsum_example.cpp)
 target_link_libraries(Direct cabanacore)
 add_executable(Ewald ewaldsum_example.cpp)
 target_link_libraries(Ewald cabanacore)
 
-if(Cabana_ENABLE_Cuda)
+if(Kokkos_ENABLE_CUDA)
   find_package(CUDA REQUIRED)
```

Comment:
Do we still need to detect `CUDA` or is `find_package(Kokkos)` pull that in?

Reply:
We do.  Kokkos does not link to Cuda FFT.